content.js

```diff
@@ -107,13 +112,21 @@ document.onkeydown = function(e) {
     let video = document.getElementById("movie_player");
     //is the video in focus, otherwise they could be typing a comment
     if (document.activeElement === video) {
-        if (e.code == hotkeys.submit) {
-            if (hasData) { // Submit!!!
+        if (e.code == Settings.hotkeys.submit) {
+            if(Settings.onekey) {
+              if (hasData) { // Submit!!!
```

Comment:
as I've said before,  this is not going to work sadly. The data can be changed by clicking on the buttons as well as hotkeys. I'd recommend checking if the submit button is visible instead of using that variable.

Reply:
You have a good point there last time I did not change it because I was hoping the global popup would be changed to settings

Reply:
There are also buttons on the player

Reply:
Is this fixed now?

Reply:
should be, you'd have to test it